docs/source/stability_policy.rst

```diff
@@ -0,0 +1,137 @@
+Stability Policy
+================
+
+The ``python-telegram-bot`` library (PTB) tries to provides a stable interface to it’s users.
+As both the user base and the complexity of the library has grown significantly since PTBs conception, we introduced below explicit stability Policy with version NEXT.VERSION.
+
+Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, the stability policy of which PTB can not influence.
+This policy hence includes some special cases for those parts.
+
+What does this policy cover?
+----------------------------
+
+Non-Bot API Functionality
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This policy includes any API or behavior that is documented in this documentation.
+This covers both the :mod:`telegram` package and the :mod:`telegram.ext` package.
+
+Bot API Functionality
+~~~~~~~~~~~~~~~~~~~~~
+
+What doesn’t this policy cover?
+-------------------------------
+
+Introduction of new features or changes of flavors of comparable behavior (e.g. the default for the used HTTP protocol version) are not covered by this policy.
+The internal structure of classes in PTB, i.e. things like the result of ``dir(obj))`` or the contents of ``obj.__dict__``, may change.
+
+Objects are in general not guaranteed to be pickleable (unless stated otherwise) and pickled objects from one version of PTB may not be loadable in future versions.
+We may provide a way to convert pickled objects from one version to another, but this is not guaranteed.
+
+-  ☐ Document pickle stability for :class:`~telegram.TelegramObject` and error classes
```

Comment:
?
Already done above

Reply:
ah no, I meant that the docs of `TelegramObject` and the error classes should state that those can in fact be pickled (we made quite an effort for that), but there might still be loading issues on (major) version changes.

Reply:
oh ok, that looked like a checkbox for us to complete

Reply:
That's exactly what it is, yes :D